sdk/storage/azure-storage-file-share/perf-resources.bicep

```diff
@@ -0,0 +1,30 @@
+param baseName string = resourceGroup().name
+param location string = resourceGroup().location
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2019-06-01' = {
+  name: '${baseName}files'
+  location: location
+  kind: 'FileStorage'
+  sku: {
+    name: 'Premium_LRS'
+  }
+
+  resource service 'fileServices' = {
+    name: 'default'
+    properties: {
+      shareDeleteRetentionPolicy: {
```

Comment:
Should delete retention policy be disabled for Blob and Datalake as well?

Reply:
I believe this bicep property only applies to file-share:

https://docs.microsoft.com/en-us/search/?terms=shareDeleteRetentionPolicy